DESCRIPTION

```diff
@@ -30,7 +30,6 @@ Depends:
     methods,
     R (>= 3.0.0)
 Imports:
-    stringr,
     Rcpp (>= 0.12.13)
```

Comment:
Need to remove `import("stringr")` from the namespace to avoid check failures.

Reply:
Gocha. Done.